falcosidekick/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ numbering uses [semantic versioning](http://semver.org).
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
 
+## 0.5.5
+
+* Add `webui.externalRedis` with `enabled`, `url` and `port` to values to set an external Redis database with RediSearch > v2 for the WebUI
+* Add `webui.redis.enabled` option to disable the deployment of the database.
+
 ## 0.5.3
```

Comment:
```suggestion
## 0.5.4
```
I forgot to change this version number after my last copy paste. Can you update it for me please.